lib/bap/bap.mli

```diff
@@ -621,12 +621,12 @@ module Std : sig
       (** Converts an 'a to a string *)
       type 'a printer = Format.formatter -> 'a -> unit
```

Comment:
In the `bap.mli` there is the same type that is defined just after this module. If you will move it above, then this definition can be removed.